### DISCUSSION TITLE
Kesehatan dan kebugaran

### DISCUSSION THREAD

**Discussion by andikctistian** [elementor/elementor#22726] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### [ https://tamvil.blogspot.com/ ](url) @ibndawood [ [ https://my.hostidn.com/aff.php?aff=10 ](https://quwashop.com/wp-admin/admin.php?page=elementor#tab-experiments) ](url) [ 150939910 ](url)
[ https://www.tamvilbeda.my.id/wp-admin/ ](url)[ https://baruslangsung.xyz ](url)
[ https://baruslangsung.xys ](url)

### Describe the solution you'd like

@media (max-width: 600px) {
   body {
      font-size: 16px;
   }
   h1 {
      font-size: 22px;
   }
   /* other styles for small screens */
}

/* styles for screens between 601px and 900px */
@media (min-width: 601px) and (max-width: 900px) {
   body {
      font-size: 18px;
   }
   h1 {
      font-size: 26px;
   }
   /* other styles for medium screens */
}

/* styles for larger screens */
@media (min-width: 901px) {
   body {
      font-size: 20px;
   }
   h1 {
      font-size: 32px;
   }
   /* other styles for large screens */
}

**Comment by theavic** [comment]
Sorry, what's the question?
